src/common/components/floating-faq/index.scss

```diff
@@ -0,0 +1,299 @@
+.help-btn {
+  position: fixed;
+  z-index: 900;
+  right: 2rem;
+  bottom: 2rem;
+  background: $dark-sky-blue;
+  color: $white;
+  border-radius: 20px 2px 20px 20px !important;
+  padding: 10px 20px 10px 10px !important;
+
+  .help {
+    display: inline-block;
+    margin-left: 0.4rem;
+    position: relative;
+    top: 1px;
+  }
+}
+
+.faq-container {
+  position: fixed;
+  z-index: 900;
+  right: 2.1rem;
+  bottom: 5.5rem;
+  height: 78.5vh;
```

Comment:
Please, _pixelize_ it . For example, on landscape it will be too small

Reply:
Hi @dkildar , On some mobile devices the fixed pixels height is more than the screen size and it is cut out by the screen. Therefore to keep it dynamic I suggested using view height instead of fixed height. I am going to change it to 75vh to keep it in a round figure. Please see the attached screenshot. I think it looks fine on the big screen. 

![image](https://user-images.githubusercontent.com/106739598/214499888-00578e66-96d9-46c8-b098-0e461d6fdbed.png)

Reply:
It make sense